src/Microsoft.TestPlatform.Common/ExtensionFramework/Utilities/TestExtensions.cs

```diff
@@ -91,6 +91,74 @@ public class TestExtensions
     /// </summary>
     internal bool AreDataCollectorsCached { get; set; }
 
+    /// <summary>
+    /// Merge two extension maps.
+    /// </summary>
+    /// 
+    /// <param name="extensionMap1">First extension map.</param>
+    /// <param name="extensionMap2">Second extension map.</param>
+    /// 
+    /// <returns>
+    /// A map representing the merger between the two input maps.
+    /// </returns>
+    internal static IDictionary<string, ISet<string>> MergeExtensionMaps(
+        IDictionary<string, ISet<string>> extensionMap1,
+        IDictionary<string, ISet<string>> extensionMap2)
+    {
+        var isExtensionMap1Valid = (extensionMap1 != null && extensionMap1.Count > 0);
```

Comment:
Please invert the logic and rename to isNullOrEmpty

Reply:
Done.